ert_shared/libres_facade.py

```diff
@@ -151,20 +151,28 @@ def observation_keys(self, key):
         else:
             return []
 
-    def gather_gen_kw_data(self, case, key, realization_index=None):
+    def gather_gen_kw_data(
+        self, case, key, realization_index=None, read_only: bool = False
+    ):
         """:rtype: pandas.DataFrame"""
```

Comment:
`:rtype:` is deprecated in favor of type hints and the docstring is missing a description of the other parameters.